app/pb_kits/playbook/pb_date_range_stacked/_date_range_stacked.scss

```diff
@@ -0,0 +1,16 @@
+.pb_date_range_stacked {
+  .pb_date_range_stacked_display {
+    display: inline-grid;
+    .pb_date_range_stacked_arrow {
+      top: 0.8125rem;
+      position: relative;
+      margin: 0 0.625rem 0;
```

Comment:
Can you use our spacing variables? https://github.com/powerhome/playbook/blob/v3.5.0/app/pb_kits/playbook/tokens/_spacing.scss
Also could this not be solved with flexbox?

Reply:
I made this example: https://jsfiddle.net/oahefj9z/12/